.husky/pre-commit

```diff
@@ -2,4 +2,11 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn precommit
-./hooks/pre-commit.nocommit
+
+# Run the linter
+./bin/quick_lint.bb
```

Comment:
After looking at it, feels like setting up lint-staged to call clj-kondo makes sense, it will be simpler to have one way to handle linting. Also that would be 1 place to handle precommit hooks, too.  The formatting can happen the same way. Notice the FE folks are calling `prettier` in the `"lint-staged"` config in `package.json`.

Reply:
+1 , it's much easier to figure out what's happening on the pre-commit hook if they're all defined the same way

Reply:
Done!